tensorflow_addons/seq2seq/attention_wrapper.py

```diff
@@ -1562,7 +1562,31 @@ def _compute_attention(
 
 
 class AttentionWrapper(tf.keras.layers.AbstractRNNCell):
-    """Wraps another RNN cell with attention."""
+    """Wraps another RNN cell with attention.
+
+    Example:
+
+    >>> batch_size = 4
+    >>> max_time = 7
+    >>> hidden_size = 32
+    >>>
+    >>> memory = tf.random.uniform([batch_size, max_time, hidden_size])
+    >>> memory_sequence_length = tf.fill([batch_size], max_time)
+    >>>
+    >>> attention_mechanism = tfa.seq2seq.LuongAttention(hidden_size)
+    >>> attention_mechanism.setup_memory(memory, memory_sequence_length)
+
+    >>> cell = tf.keras.layers.LSTMCell(hidden_size)
+    >>> cell = tfa.seq2seq.AttentionWrapper(
+    ...     cell, attention_mechanism, attention_layer_size=hidden_size)
+    >>>
+    >>> inputs = tf.random.uniform([batch_size, hidden_size])
+    >>> state = cell.get_initial_state(inputs)
+    >>>
+    >>> outputs, state = cell(inputs, state)
```

Comment:
I think the state can be skipped since the cell will be default call cell.get_initial_state() if the state is not passed in the call().

Reply:
The state argument [is not optional](https://github.com/tensorflow/addons/blob/v0.11.2/tensorflow_addons/seq2seq/attention_wrapper.py#L1949) for this cell, so I think we should keep it.

Reply:
oh, you are right. The state is optional for lstm_cell, but not for the wrapper.